dbt/adapters/snowflake/column.py

```diff
@@ -1,11 +1,16 @@
 from dataclasses import dataclass
+from typing import Dict, ClassVar
 
 from dbt.adapters.base.column import Column
 from dbt.exceptions import DbtRuntimeError
 
 
 @dataclass
 class SnowflakeColumn(Column):
+    TYPE_LABELS: ClassVar[Dict[str, str]] = {
```

Comment:
The parent/default `Column.TYPE_LABELS` has `STRING -> TEXT`. I haven't included it here because these are true aliases on Snowflake.